firewall.conf

```diff
@@ -15,9 +15,9 @@ iptables -A INPUT -p tcp --dport ssh -j ACCEPT
 # Allow for all traffic from anywhere for the following services
 iptables -A INPUT -p tcp --dport 80 -j ACCEPT          # nginx
 iptables -A INPUT -p tcp --dport 443 -j ACCEPT         # nginx (SSL)
-iptables -A INPUT -p tcp --dport 2812 -j ACCEPT        # monit dashboard
+#iptables -A INPUT -p tcp --dport 2812 -j ACCEPT        # monit dashboard
```

Comment:
I'd, at least, disable the dashboard buttons (without hiding them).
Otherwise, if you click, you get ERR_CONNECTION_TIMED_OUT.